### DISCUSSION TITLE
Silent mode

### DISCUSSION THREAD

**Discussion by Ameb** [QIDITECH/QIDI_Q1_Pro#64] [Ideas]
Currently under speed adjustment while printing there is a "Mute" switch which i think all it does is set speed to 50% (`M220 S50`).
I think it would be possible to reduce the stepper noise as other printers do. I found this plugin but have no idea how feasible it would be to use it with the Q1 Pro klipper: https://github.com/andrewmcgr/klipper_tmc_autotune

I think it would be possible right now to implement a Silent Mode using stealthChop mode. [Klipper's documentation](https://www.klipper3d.org/TMC_Drivers.html) describe how to enable it. At the same time I understand max velocity and acceleration would need to be reduced.

**Comment by CChen616** [comment]
We tested the older version of the auto_tune plugin, but the effects were not noticeable, and there was noise in the Z-axis in idle. The new version of auto_tune is not compatible with Klipper versions below 0.12.0 and is not suitable for the Q1.

Also tested the stealthChop mode, but the results were not good and even had negative effects making the noise louder. Besides, switching during printing caused defects.
